api/src/routes/user.ts

```diff
@@ -91,5 +100,25 @@ export const userRoutes: FastifyPluginCallbackTypebox = (
     }
   );
 
+  fastify.post('/user/user-token', async req => {
```

Comment:
I am correct in this?

```suggestion
  // TODO(Post-MVP): POST -> PUT
  fastify.post('/user/user-token', async req => {
```